### PR TITLE
Dotted circle

### PR DESCRIPTION
**Pull request by RosaWagner**
To reflect on mark attachment requirements in Docs
cc @vv-monsalve 

Will update db and lists when confirmed.

### CODE REVIEW

**Comment by m4rc1e** [comment]
I haven't really encountered the dotted circle in Latin fonts so I'm wondering if it's overkill to include it in Latin Plus?

File: GF_glyphsets/Latin/glyphs/GF_Latin_African.glyphs [hunk 1]
```diff
@@ -1790,6 +1790,16 @@ width = 600;
...
+glyphname = Tcedilla;
+layers = (
+{
+layerId = m01;
+width = 600;
+}
+);
+unicode = 354;
+},
+{
```
Comment (m4rc1e) [inline GF_glyphsets/Latin/glyphs/GF_Latin_African.glyphs:1802]: Do we want Tcedilla in here?

**Comment by vv-monsalve** [comment]
> I haven't really encountered the dotted circle in Latin fonts so I'm wondering if it's overkill to include it in Latin Plus?

But still, we are receiving the WARN. Otherwise, it could end up like one of the always ignored Warns
> ⚠ WARN: Ensure dotted circle glyph is present and can attach marks. ([com.google.fonts/check/dotted_circle](https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/dotted_circle))
⚠ WARN No dotted circle glyph present [code: missing-dotted-circle]

**Comment by vv-monsalve** [comment]
I wonder now if this filter to [Add dotted circle](https://github.com/googlefonts/ufo2ft/pull/593) should be included by default when using Builder.

**Comment by vv-monsalve** [comment]
> Does it worth the trouble? If not I would only add it to the phonetic sets (where it is needed).

If it would be too complicated at least the second option would be ok.

> If we think that it would actually be good to completely go along with that FB, and make it a convention to all designers; then let's make the builder generating it by default.

Yes, the filter is already there

> I personally think the check should be skipped for all scripts expect the one where it is really needed.

But even in those cases it is being skipped https://github.com/google/fonts/pull/4850#issuecomment-1170888439. And again, the filter seems to be already at hand to solve this.

**Pull request closed without merging**